examples/large_models/vllm/Readme.md

```diff
@@ -13,3 +19,25 @@ vLLM [EngineArgs](https://github.com/vllm-project/vllm/blob/258a2c58d08fc7a24255
 
 * Sampling parameters for text generation:
 vLLM [SamplingParams](https://github.com/vllm-project/vllm/blob/258a2c58d08fc7a242556120877a89404861fbce/vllm/sampling_params.py#L27) is defined in the JSON format, for example, [prompt.json](lora/prompt.json).
+
+### Distributed Inference
+All examples can be easily distributed over multiple GPUs by enabling tensor parallelism in vLLM.
+To enable distributed inference the following additions need to made to the model-config.yaml of the examples where 4 is the number of desired GPUs to use for the inference:
+
+```yaml
+# TorchServe frontend parameters
+...
+parallelType: "custom"
```

Comment:
What does `parallelType` mean? and what is `custom` . What are the other options?
We should perhaps talk about these options in a README under docs?

Reply:
parallelType was already there, options are [tp, pp, and pptp](https://github.com/pytorch/serve/blob/079ff7b5d31d79627fa73d247c4f86a12893f8c5/examples/large_models/tp_llama/model-config.yaml#L6). Its basically the torchrun integration. "Custom" lets users customize the way they do parallelism. Will see if there is more doc on this and link here or add an issue to improve docs on parallelType.